src/Blazor.Extensions.SignalR/Blazor.Extensions.SignalR.csproj

```diff
@@ -1,17 +1,25 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <Title>Blazor Extensions SignalR</Title>
     <Description>SignalR Core support for ASP.NET Core Blazor.</Description>
   </PropertyGroup>
 
   <PropertyGroup>
-    <OutputType>Library</OutputType>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <RestoreAdditionalProjectSources>

```

Comment:
Why this? Preview4 is on nuget.org

Reply:
You're right, I merged it from the blazorlib template.  Fixed it in the latest push.